gauge-ts/src/loaders/ImplLoader.ts

```diff
@@ -11,10 +11,10 @@ type ModuleType = {
 };
 
 export class ImplLoader {
-  private paramterParsignChain: ParameterParsingChain;
+  private parameterParsignChain: ParameterParsingChain;
```

Comment:
Sorry to be irritating. But should this be
```suggestion
  private parameterParsingChain: ParameterParsingChain;
```